Cannot find module 'chalk'

In installing process users got below message:
```js
> node ./preinstall.js
module.js:471
    throw err;
    ^
Error: Cannot find module 'chalk'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/travis/build/msudgh/icu-loader/preinstall.js:3:15)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

--- a/package.json
+++ b/package.json
@@ -36,7 +36,6 @@
   },
   "devDependencies": {
     "babel-loader": "^6.2.10",
-    "chalk": "^2.0.1",
     "jest-cli": "^18.0.0",
     "sloc": "^0.1.11",
     "snazzy": "^5.0.0",

--- a/preinstall.js
+++ b/preinstall.js
@@ -1,5 +1,3 @@
 'use strict'
 
-const chalk = require('chalk')
-
-console.log(chalk.bold.red('⚠ Warning:'), `Since this version that you installed the syntax of package has been changed to Promise way, to figure out check the documentation of library. (Backwards incompatible)`)
+console.log('⚠ Warning: Since this version that you installed, the syntax of package has been changed to Promise way, to figure out new changes check out the documentation of package. (Backwards incompatible)')

